src/components/timeToString.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect, useRef } from 'react';
+import { useSelector } from 'react-redux';
+import { format, parseISO } from 'date-fns';
+
+export const timeToString = (value) => {
+    const isMounted = useRef(null);
+	const [locale, setLocale] = useState('');
+	const language = useSelector(state => state.language.language)
+	
+	// Sets isMounted indicator to true
+	// when component has been mounted (componentDidMount analogue)
+	// and sets isMounted to false before component 
+	// is unmounted (componentWillUnmount analogue)
+	useEffect(() => {
```

Comment:
This function shouldn't use any react hooks b/c in that case it can be used only as React Component.
We need only value, not a component.

Reply:
I second that